sandy/sections/mf2.py

```diff
@@ -0,0 +1,860 @@
+    [{'PPI': 1.0, 'L': 0.0, 'SCH': 0.0, 'BND': 0.0, 'APE': 0.0, 'APT': 0.0}, 
+     {'PPI': 2.0, 'L': 1.0, 'SCH': 0.5, 'BND': 0.0, 'APE': 0.54373, 'APT': 0.54373}]
+    
+    
+    Resonance parameters of the Uranium 235
+    LRU = 2 LRF = 2
+    >>> tape = sandy.get_endf6_file("jeff_33", "xs", 922350)
+    >>> dic = sandy.read_mf2(tape, 9228)
+    >>> print (dic["NIS"]["ISO"][92235]["NER"][(2250.0, 46200.0)]['L'][0]["J"][0]["RES_PAR"][0])
+    {'ES': 2250.0, 'D': 1.058, 'GX': 0.0, 'GNO': 0.000107789, 'GG': 0.038513, 'GF': 0.40102}
+    
+    """
+    df = tape._get_section_df(mat, mf, mt)
+    out = {
+            "MAT": mat,         
+            "MF": mf,
+            "MT": mt,
+            }
+    i = 0
+    I = {}
+    C, i = sandy.read_cont(df, i)
+    add = {
+            "ZA": C.C1,             #designation for an isotope
+            "AWR": C.C2,            #AWR is defines as the ratio of the mass of the material to that of the neutron
+            "NIS": C.N1,            #NIS is the number of isotopes in the material
+            }
+    NIS = int(C.N1)
+    I.update(add)
+    NER = int(C.N1)
+    LFW = int(C.L2)
+    out.update({"INTRO": I})
+    P = {}
+    for l in range(NIS): 
+        M = {}
+        NER1 = {}
+        ISO = {}
+        C, i = sandy.read_cont(df, i)
+        header1 = {
+                "ZAI": C.C1,            #designation for an isotope
+                "ABN": C.C2,            #Abundance of an isotope in the material 
+                "LFW": C.L2,            #indication whether average fission wifths are given in the unresolbed resonance region
+                } 
+        NER = int(C.N1)
+        LFW = int(C.L2)
+        ZAI = int(C.C1)
+        M.update(header1)
+        dico = {}
+        for j in range(NER):
+            info = {}
+            C, i = sandy.read_cont(df, i)
+            header2 = {
+                    "EL": C.C1,             #Lower limit for an energy range
+                    "EH": C.C2,             #Upper limit for an energy range
+                    "LRU": C.L1,            #Flag indicating whether this energy range contains data for resolved or unresolved resonance parameters:
+                    "LRF": C.L2,            #Flag indicating which representation has been used for the energy range. 
+                    "NRO": C.N1,            #Flag designating possible energy dependence of the scattering radiu
+                    "NAPS":C.N2,            #Flag controlling the use of the two radii
+                    }
+            EL = C.C1
+            EH = C.C2
+            LRU = int(C.L1)
+            LRF = int(C.L2)
+            NRO = int(C.N1)
+            if LRU == 0:
+                LRU0 = {}
+                LRU0.update(header2)
+                C, i = sandy.read_cont(df, i)
+                add = {
+                        "SPI": C.C1,        #Spin, I, of the target nucleus.
+                        "AP": C.C2,         #Scattering radius in units of 10e-12cm.
```

Comment:
Is this 0? If it is always 0 we should remove it

Reply:
it is not always equal to 0